Scripts/Python/plasma/PlasmaGame.py

```diff
@@ -41,79 +41,150 @@
 
  *==LICENSE==* """
 
+# NOTE: This stub file was generated automatically from Plasma's Python interface.
+# Do not edit this file manually.
+# To change any of the docstrings or function signatures,
+# edit the corresponding C++ glue code in pfPython.
+# If the Python interface has changed, regenerate these stubs
+# by running the following call in the in-game Python console:
+# >>> __import__("generate_stubs").run()
+
 from __future__ import annotations
 from typing import *
 
 class ptGameCli:
     """Abstract bass class for legacy game clients."""
 
-    gameID: int = ...
+    gameID: int
+    """The ID of the game instance on the server."""
+
+    handler: Any
+    """The game event handler."""
+
+    isLocallyOwned: bool
+    """Whether or not we are the owner of this game instance."""
+
+    ownerID: int
+    """The ID of the player who owns this game instance."""
+
+    def __init__(self):
+        """None"""
+        pass
+
+    def leaveGame(self) -> None:
+        """Explicitly ask the server to allow us to leave the game."""
+        pass
+
+class ptGmBlueSpiral(ptGameCli):
+    """Legacy blue spiral game client."""
+
+    gameID: int
     """The ID of the game instance on the server."""
 
-    handler: Any = ...
+    handler: Any
     """The game event handler."""
 
-    isLocallyOwned: bool = ...
+    isLocallyOwned: bool
     """Whether or not we are the owner of this game instance."""
 
-    ownerID: int = ...
+    ownerID: int
     """The ID of the player who owns this game instance."""
 
+    def __init__(self):
+        """None"""
+        pass
+
+    def hitCloth(self):
+        """Request for the server to hit a specific cloth index and validate the correct sequence of cloth inputs."""
+        pass
```

Comment:
The binding itself must be missing the `Params: ...` line

Reply:
I've added type annotations and parameter names for all the `ptGmBlueSpiral` methods now.